app/src/main/java/sopt/org/millie/presentation/search/library/SearchLibraryScreen.kt

```diff
@@ -0,0 +1,78 @@
+package sopt.org.millie.presentation.search.library
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import sopt.org.millie.R
+import sopt.org.millie.presentation.search.component.SearchTitle
+import sopt.org.millie.presentation.search.component.library.LibraryItem
+import sopt.org.millie.presentation.search.model.SearchLibraryModel
+
+@Composable
+fun SearchLibraryScreen(
+    libraryList: List<SearchLibraryModel>,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .padding(top = 23.dp)
+            .padding(horizontal = 20.dp),
+    ) {
+        SearchTitle(
+            title = "서재",
+            count = libraryList.size,
+            modifier = Modifier.padding(start = 1.dp),
+        )
+
+        Spacer(modifier = Modifier.height(13.dp))
+
+        Row(
+            modifier = modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.spacedBy(19.dp),
```

Comment:
```suggestion
            horizontalArrangement = Arrangement.SpaceBetween,
```
이거 공간이 너무 남아서...........못생겻어도 그냥 SpaceBetween 줍시다..

Reply:
넹..ㅎㅎ